src/SelectionInfoView.tsx

```diff
@@ -1,34 +1,58 @@
 import * as React from "react";
 
-import { MarkerFileData } from "./DataTypes";
+import { MarkerFileData, MarkerFrame } from "./DataTypes";
+import { computeAngle} from "./Calculations";
```

Comment:
It looks like the MarkerFrame import isn't needed any more, so line 3 can be reverted to how it was before

Reply:
Oh yeah, fixed.